docs/source/aryn_cloud/aryn_partitioning_service.md

```diff
@@ -118,3 +118,45 @@ You can use multiple options at the same time like in the example below:
 export ARYN_API_KEY="PUT API KEY HERE"
 curl "https://api.aryn.cloud/v1/document/partition" -o "document.json" -H "Authorization: Bearer $ARYN_API_KEY" -F "pdf=@document.pdf" -F "options={\"extract_table_structure\": true, \"threshold\": 0.2}";
 ```
+
+## Output Format
+
+The output of the Aryn Partitioning Service is json.
+
+```text
+{ "status": in-progress updates,
+  "error": any errors encountered,
+  "elements": a list of elements }
+```
+
+### Element Format
+
+{
+```text
+"type": one of the element types below,
+```
+
+Element Types: "N/A","Caption","Footnote","Formula","List-item","Page-footer","Page-header","Picture","Section-header","Table","Text","Title"
+
+```text
+"bbox": coordinates of the bounding box around the element contents,
+```
+Takes the format `[x1, y1, x2, y2]` where it is a proportion of how far down the screen the element is.
+```text
+"properties":
+    { "score": confidence (between 0 and 1, with 1 being the most confident),
+      "page_number": 1-indexed page number the element occurs on },
+```
+
+```text
+"text_representation": text associated with this element,
+```
+
+Text elements contain ‘\n’ when the text includes a line return
+
+When `extract_images` is set to True, Pictures include a binary_representation tag which contains a base64 encoded ppm image file of the pdf cropped to the bounds of the detected image. When `extract_images` is false, the bounding box of the Picture is still returned.
```

Comment:
Should we explain the table output as well, or is that pending on Ben?

Reply:
This is pending on @bsowell